src/main/java/org/semux/core/TransactionResult.java

```diff
@@ -72,85 +92,99 @@
          */
         INVALID_DELEGATE_NAME,
 
+        /**
+         * Invalid burning address.
+         */
+        INVALID_BURNING_ADDRESS,
+
         /**
          * Invalid delegate burn amount.
          */
-        INVALID_DELEGATE_BURN_AMOUNT,
+        INVALID_BURNING_AMOUNT,
+
+        /**
+         * The DELEGATE operation is invalid.
+         */
+        INVALID_DELEGATING,
+
+        /**
+         * The VOTE operation is invalid.
+         */
+        INVALID_VOTING,
 
         /**
-         * Transaction failed.
+         * The UNVOTE operation is invalid.
          */
-        FAILED
+        INVALID_UNVOTING;
+
+        public boolean isSuccess() {
+            return this == SUCCESS;
+        }
+
+        public boolean isFailure() {
+            return this == FAILURE;
+        }
+
+        public boolean isAccepted() {
```

Comment:
It's strange to have isAccepted() and isRejected()

Would suggest at minimum to make isAccepted() return !isRejected() or vice versa, rather than have 2 separate implementations for maintenance purposes.